/wbc/joint_position/controller:
  ros__parameters:
    control_rate: 250.0
    joint_names: ["joint_0","joint_1","joint_2","joint_3","joint_4","joint_5","joint_6"]
    p_gain: [10.0,10.0,10.0,10.0,10.0,10.0,10.0]
    d_gain: [1.1,1.1,1.1,1.1,1.1,1.1,1.1]
    ff_gain: [0.0,0.0,0.0,0.0,0.0,0.0,0.0]
    max_control_output: [5.0,5.0,5.0,5.0,5.0,5.0,5.0]
    dead_zone: [0.0,0.0,0.0,0.0,0.0,0.0,0.0]
